packages/dd-trace/src/log/index.js

```diff
@@ -15,7 +15,12 @@ const config = {
   logLevel: 'debug'
 }
 
+const mute = new LogConfig(false)
+
 const log = {
+  LogConfig,
+  MUTE: mute,
```

Comment:
The word "mute" will easily be misinterpreted, as we're not muting the log in general, but just not sending it to telemetry. A more descriptive name would be:

```suggestion
  NO_TELEMETRY: mute,
```